Sriram/Reg.html

```diff
@@ -13,7 +13,7 @@ <h2><strong>Register</strong></h2>
       First name:
       <input type="text" name="firstname" placeholder="First name" autocomplete="username" required/>
       Last name:
-      <input type="text" name="lastname" placeholder="Last name" autocomplete="on" /><br /><br />
+      <input type="text" name="lastname" placeholder="Last name" autocomplete="username" /><br /><br />
```

Comment:
That's wrong. Username is a login name, not a person's first and last name. Read through the link I gave.